### ISSUE TITLE
high resource usage when parallelizing

### ISSUE THREAD

**Issue by jjuod** [william-denault/WaveletScreaming#2]
Parallel version seems to eat lots of RAM, at least at higher resolutions. For now, default setting changed to `para=FALSE` and parallelization is to be used at user's own risk. Would be good to add some max input size/resolution recommendations for parallel mode.

**Comment by william-denault** [comment]
Agreed I will check that, and may be that could be done by the user. Or we could propose mutilple paralelmisation, i.e apply parApply or normal

**Comment by jjuod** [comment]
In addition: some systems appear to have linear algebra libraries which already implement parallelization. This means that many CPUs are reserved, but the calculations are relatively short and the overhead is not worth it. Recommend `RhpcBLASctl`, to limit BLAS threads to 1 or 2, as in commit 8124d2c78416a09bff558d35b0ffba6a5b1b5983
